### DISCUSSION TITLE
tun2socks traffic loops with local proxy of qv2ray

### DISCUSSION THREAD

**Discussion by MiliAxe** [xjasonlyu/tun2socks#180] [Q&A]
I'm trying to tunnel all of my traffic via the socks5 proxy provided by my qv2ray program and I have followed every step in the quick start
part and even tried to delete the interface parameter but when I start tun2socks via `tun2socks -device tun0 -proxy socks5://127.0.0.1:1089`, the program starts looping and uses all of my CPU. 

What step I have done:
`ip tuntap add mode tun dev tun0`
`ip addr add 198.18.0.1/15 dev tun0`
`ip link set dev tun0 up`
`ip route del default`
`ip route add default via 198.18.0.1 dev tun0 metric 1`
`ip route add default via 192.168.1.1 dev wlan0 metric 10`
`tun2socks -device tun0 -proxy socks5://127.0.0.1:1089`

Thanks for the great program.

**Comment by chenwanqq** [comment]
same.

**Comment by SaeedDev94** [comment]
https://medium.com/@SaeedPooyanfar/use-socks-proxy-as-vpn-in-linux-d31b1cce44e0
